tox.ini

```diff
@@ -4,6 +4,7 @@ envlist =
 	py{3.7,3.8,3.9,3.10}-django3.2
 	py{3.8,3.9,3.10,3.11}-django4.1
 	py{3.8,3.9,3.10,3.11}-django4.2
```

Comment:
Django 4.2.7 will support Python 3.12 https://forum.djangoproject.com/t/django-4-2-lts-compatibility-with-python-3-12/24508/3

Reply:
Hmmmmm
> if we will be able to properly test it with Python 3.12 before the release date (November, 1st)

https://docs.djangoproject.com/en/4.2/releases/4.2.7/
There is no reference to this in the release notes currently and as the release is tomorrow, I doubt it will make it in.

Reply:
You are right, I took "I see a lot of changes have been merged to the 4.2.x branch but not official support yet." for granted  that the support would be there for 4.2.7. Thanks for pointing that out!